### PR TITLE
remove sinon-timers

### PR DESCRIPTION
**Pull request by adambrunner**
Fix for https://github.com/kmees/karma-sinon-chai/issues/34

### CODE REVIEW

**Comment by icebob** [comment]
Great!

File: index.js [hunk 1]
```diff
@@ -13,30 +13,27 @@ var endsWith = function(substr) {
...
-}
+};
 
 var framework = function(files) {
-  var isDuplicate = _isDuplicate.bind(this, files)
+  var isDuplicate = _isDuplicate.bind(this, files);
 
   /* Lolex */
-  var lolexPath = path.resolve(require.resolve('lolex'), '../../lolex.js');
+  var lolexPath = path.dirname(require.resolve('lolex/packages.json')) + '/lolex.js';
```
Comment (sirlancelot) [inline index.js]: Should probably be `package.json` (-s)

File: index.js [hunk 2]
```diff
@@ -13,30 +13,27 @@ var endsWith = function(substr) {
...
+  var sinonPath = path.dirname(require.resolve('sinon/packages.json')) + '/pkg/sinon.js';
   if (!isDuplicate(sinonPath)) {
-    files.unshift(pattern(sinonTimersPath));
     files.unshift(pattern(lolexPath));
     files.unshift(pattern(sinonPath));
   }
 
   /* Chai */
-  var chaiPath = path.resolve(require.resolve('chai'), '../chai.js');
+  var chaiPath = path.dirname(require.resolve('chai/packages.json')) + '/chai.js';
```
Comment (sirlancelot) [inline index.js]: Should probably be `package.json` (-s)

**Comment by ZuBB** [comment]
@kmees what do you think about this MR?

**Comment by davezuko** [comment]
Would like to see this merged. I've posted an [issue](https://github.com/kmees/karma-sinon-chai/issues/37) concerning the state of this project.

**Comment by ZuBB** [comment]
thanks a lot @adambrunner  for 42818f5

**Pull request merged**